src/setupProxy.js

```diff
@@ -12,19 +12,19 @@ module.exports = (app) => {
   // );
 
   // @TRAIN
-  app.use(
-    '/api',
-    createProxyMiddleware({ target: 'https://360ApiTrain.gordon.edu/', changeOrigin: true }),
-  );
+  // app.use(
+  //   '/api',
+  //   createProxyMiddleware({ target: 'https://360ApiTrain.gordon.edu/', changeOrigin: true }),
+  // );
+  // app.use(
+  //   '/token',
+  //   createProxyMiddleware({ target: 'https://360ApiTrain.gordon.edu/', changeOrigin: true }),
+  // );
+
+  // @LOCALHOST
+  app.use('/api', createProxyMiddleware({ target: 'http://localhost:1995/', changeOrigin: true }));
   app.use(
     '/token',
-    createProxyMiddleware({ target: 'https://360ApiTrain.gordon.edu/', changeOrigin: true }),
+    createProxyMiddleware({ target: 'http://localhost:1995/', changeOrigin: true }),
   );
```

Comment:
Let's undo the changes made to this file.

Reply:
done